validator_client/src/cli.rs

```diff
@@ -367,6 +367,16 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                     constructed by builders, regardless of payload value.")
                 .takes_value(false),
         )
+        .arg(
+            Arg::with_name("disable-slashing-protection-web3signer")
+                .long("disable-slashing-protection-web3signer")
+                .help("Disable Lighthouse's slashing protection for all web3signer keys. This can \
+                       reduce the I/O burden on the VC but is only safe if slashing protection \
+                       is enabled on the remote signer and is implemented correctly. DO NOT ENABLE \
+                       THIS FLAG UNLESS YOU ARE CERTAIN THAT SLASHING PROTECTION IS ENABLED ON \
+                       THE REMOTE SIGNER. IT IS OFF BY DEFAULT, AND YOU WILL GET SLASHED IF YOU \
+                       USE THIS FLAG WITHOUT ENABLING IT.")
```

Comment:
```suggestion
            Arg::with_name("disable-slashing-protection-web3signer")
                .long("disable-slashing-protection-web3signer")
                .help("Disable Lighthouse's slashing protection for all web3signer keys. This can \
                       reduce the I/O burden on the VC but is only safe if slashing protection \
                       is enabled on the remote signer and is implemented correctly. DO NOT ENABLE \
                       THIS FLAG UNLESS YOU ARE CERTAIN THAT SLASHING PROTECTION IS ENABLED ON \
                       THE REMOTE SIGNER. IT IS OFF BY DEFAULT, AND YOU WILL GET SLASHED IF YOU \
                       USE THIS FLAG WITHOUT ENABLING IT.")
                .takes_value(false),
```

Reply:
^ I think we should explicitly disallow value given we're not parsing it. This is to prevent users from doing something like `disable-slashing-protection-web3signer false` when they actually meant to enable slashing protection.